src/landscape/components/LandscapeView.js

```diff
@@ -360,8 +360,9 @@ const LandscapeView = () => {
                 <CardActions sx={{ paddingTop: 0 }}>
                   <Button
                     variant="outlined"
-                    component={RouterLink}
-                    to={`/landscapes/${landscape.slug}/boundaries`}
+                    onClick={() =>
```

Comment:
Issue: There is a component called `RouterButton` ideally that should be used here